### PR TITLE
Add pluggable ldcache backend

### PR DESCRIPTION
**Pull request by acoburn**
Addresses: https://jira.duraspace.org/browse/FCREPO-2119

### CODE REVIEW

File: fcrepo-service-ldcache-file/src/main/resources/OSGI-INF/blueprint/ldcache-service.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (awoods) [inline fcrepo-service-ldcache-file/src/main/resources/OSGI-INF/blueprint/ldcache-service.xml]: The pattern you have established with the other modules is to name this file: `blueprint.xml`.
Is there any technical reason for the `blueprint.xml` naming convention? or for changing it for this module?

File: fcrepo-service-ldcache-file/pom.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    </parent>
+
+    <artifactId>fcrepo-service-ldcache-file</artifactId>
+    <packaging>bundle</packaging>
+
+    <name>File backend for the Linked Data Caching Service</name>
+    <description>A file backend for the Linked Data Caching Service</description>
+
+    <properties>
+      <!--<osgi.export.packages>org.fcrepo.camel.service.ldcache.file;version=${project.version}</osgi.export.packages>-->
```
Comment (awoods) [inline fcrepo-service-ldcache-file/pom.xml]: Any reason not to remove this commented line?

File: fcrepo-service-ldcache-file/src/main/resources/OSGI-INF/blueprint/ldcache-service.xml (hunk 1 above)
Comment (acoburn) [inline fcrepo-service-ldcache-file/src/main/resources/OSGI-INF/blueprint/ldcache-service.xml]: No, for our purposes, it just needs to have an xml extension and be in the `OSGI-INF` directory.

File: fcrepo-service-ldcache-file/pom.xml (hunk 2 above)
Comment (acoburn) [inline fcrepo-service-ldcache-file/pom.xml]: good catch, I will remove it in the next commit

**Comment by acoburn** [comment]
Merged via https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/commit/42562bb83eb0a523b8577ab8c6ed19958120b319 and https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/commit/e86b7ad4f8ca41ba2d1d0c6cadd6676d8bae418b

**Pull request closed without merging**